### PR TITLE
crystal support for 1.0.0

### PR DESCRIPTION
**Pull request by dare892**

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: granite
 
-version: 0.22.0
+version: 1.0.0
 
-crystal: 0.35.0
+crystal: 1.0.0
```
Comment (robacarp) [inline shard.yml:5]: Does the version specifier need to be more specific here? The semantics of the crystal version comparison [aren't quite what most people expect](https://github.com/crystal-lang/shards/blob/master/docs/shard.yml.adoc)

File: shard.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name: granite
 
-version: 0.22.0
+version: 1.0.0
```
Comment (Blacksmoke16) [inline shard.yml:3]: I would ensure the Amber team actually wants this to be a `1.0.0` release.

Comment (dare892) [reply shard.yml:3]: @elorest ?

File: shard.yml (hunk 1 above)
Comment (dare892) [reply shard.yml]: updated, thanks

File: shard.yml (hunk 2 above)
Comment (robacarp) [reply shard.yml:3]: Good question @Blacksmoke16. I would say no, I don't think that's warranted. A 0.23 is what I'd expect.

**Comment by Blacksmoke16** [comment]
@elorest I hope you're making another PR before cutting a release. Otherwise you're going to end up with Granite 1.0.0.

**Comment by Blacksmoke16** [comment]
NVM, I see you fixed it.

**Pull request merged**